packages/components/package.json

```diff
@@ -31,7 +31,7 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components/tree/master/packages/components#readme",
     "peerDependencies": {
-        "@patternfly/react-table": ">=4.5.7",
+        "@patternfly/react-table": ">=4.19.5",
```

Comment:
If you want to update the version of `react-table` you'll have to do it in root package.